runtime/memory-map/src/lib.rs

```diff
@@ -337,6 +337,28 @@ pub extern "C" fn memory_map_region_get_prot(map: &MemoryMap, needle: Range) ->
 /** memory_map_region_get_prot found no or multiple protections in the given range */
 pub const PROT_INDETERMINATE: u32 = 0xFFFFFFFFu32;
 
+const PKEY_MULTIPLE: u8 = 255;
```

Comment:
Do we need to keep these in sync with anything e.g. some libc function return value or are they arbitrary?

Reply:
`PROT_INDETERMINATE` just needs to be distinguishable from any valid combination of `PROT_` flags, and `PKEY_MULTIPLE` just needs to be outside the range of valid pkeys (0-15). They need to be synchronized between the Rust (this file) and the C usage, but cbindgen does that for us.